ksql-engine/src/main/java/io/confluent/ksql/schema/registry/SchemaRegistryUtil.java

```diff
@@ -37,35 +41,51 @@ public static void cleanUpInternalTopicAvroSchemas(
       final String applicationId,
       final SchemaRegistryClient schemaRegistryClient
   ) {
-    getInternalSubjectNameSet(applicationId, schemaRegistryClient)
-        .forEach(subject -> tryDeleteSubject(applicationId, schemaRegistryClient, subject));
+    getInternalSubjectNames(applicationId, schemaRegistryClient)
```

Comment:
nice. I too do not like variables / functions that include contain types in them!